ui/app/AppLayouts/Profile/popups/TokenSettingsModal.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+
+import utils 1.0
+
+import StatusQ.Controls 0.1
+
+import shared.popups 1.0
+import "../panels"
+import "../stores"
+
+// TODO: replace with StatusModal
+ModalPopup {
```

Comment:
Can we use StatusModal already please> I hope I am not being annoying :(

Reply:
Done